features/font-size.yml

```
@@ -0,0 +1,11 @@
+name: font-size
+description: The `font-size` CSS property sets the text size.
```

The spec says height, which feels like a nice way to dodge the "size sets size" circularity here.

```suggestion
description: The `font-size` CSS property sets the text height.
```